src/app/config.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class Constants {
+  BEURL = 'https://nsp-milk.herokuapp.com/api/';
+  CUSTOMER = 'customer';
+  CUSTOMERS = 'customers';
+  INVOICE = 'invoice/';
+  UPDATEINVOICE = 'update_invoice';
+  UPDATEEXTRAANDMINUSAMOUNT = '/extra-and-minus-milk';
```

Comment:
Words can be separated with underscore to enhance the readability. Also the variable name can be more appropriate like update_amount or calculate_amount.

Reply:
I will update feature development